django_sorcery/db/mixins.py

```diff
@@ -48,13 +48,14 @@ def clean_fields(self, exclude):
         for name, field in self._get_properties_for_validation().items():
             is_blank = getattr(self, name) is None
             is_nullable = field.column.nullable
+            is_pk = field.column.primary_key
             is_defaulted = field.column.default or field.column.server_default
             # skip validation if field is blank and either when field is nullable
             # so blank value is valid or field has either local or server default value
             # since we assume default value will pass validation
             # since default values are assigned during flush which as after
             # which validation is verified
-            is_skippable = is_blank and (is_nullable or is_defaulted)
+            is_skippable = is_blank and (is_nullable or is_defaulted or not is_pk)
```

Comment:
should that be `or is_pk`? this will skip when the field is blank and is non-pk. I thought the issue was we needed to skip pk fields since they are not necessarily defaulted and not nullable but we cant run skip validation when relations set the pk....